addons/website_forum/models/forum_forum.py

```diff
@@ -138,11 +138,18 @@ def _compute_teaser(self):
 
```

Comment:
Commit message is a bit strange, took me some time to figure out that you were mainly justifying picking saas-16.3.

Would say it really does not matter, just explain why you change what you change and leave the version pick aside.